lib/astarte/client/appengine/stats.ex

```diff
@@ -0,0 +1,34 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2021 SECO Mind
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.Client.AppEngine.Stats do
+  alias Astarte.Client.{APIError, AppEngine}
+
+  def get(%AppEngine{} = client) do
```

Comment:
call it `get_devices_stats`, since we will probably introduce addidional stats in the future